README.md

```diff
@@ -1 +1,103 @@
+
+We want web component development to be accessible and approachable for everyone, regardless of your background or previous experience. Therefore, our recommendations aim to be easy to use, be ready to use, and provide that "it just works" developer experience we all crave for the various aspects of web component development.
+
+We strongly believe that staying close to browser standards will be the best long term investment for your code. It is the basis for all our recommendations, and it means that sometimes we will not recommend a popular feature or functionality. It also means we can be faster to adopt and recommend new browser standards.
+
+
+## Why Web Components
+
+In the last several years, the component-based model for web application development was popularized, and the JavaScript community blossomed with a wide variety of libraries and approaches. Work on standardizing the web’s native component model began at Google in 2012, and after several years of open development, was successfully implemented across all major browsers in 2019. At the time of writing, over 10% of all page views contain web components.
+
+We believe web components provide a standards-based solution to problems like reusability, interopability, and encapsulation. Furthermore, we believe using the browser's native component model will increase the longevity of your application. The web has an extremely strong tradition of backwards-compatibility, as standards bodies have consistently gone out of their way to maintain legacy APIs. 
+
+At `open-wc` you'll find anything you might need to get started developing web components.
+
+## Developing
+
+In our [developing](https://open-wc.org/developing/) section you'll find anything you need to know about writing your code; from practical code demos, to accessibility, to mixins, to tutorials and blog posts.
+
+We have a wide range of interactive demo's available to help you get started writing code quickly:
+
+[![demos](https://i.imgur.com/CHs2d9a.png)](https://open-wc-lit-demos.stackblitz.io/)
+
+If you want to learn more about web components first, we recommend the following blog posts to give you a solid basis to start developing web components:
+
+- [Lets Build Web Components!](https://dev.to/bennypowers/lets-build-web-components-part-1-the-standards-3e85) by Benny Powers
+- [Web Components: from Zero to Hero]() by Pascal Schilp
+
+## Testing
+
+Not only do we provide a [testing setup](https://open-wc.org/testing/testing-karma.html) with Karma, Browserstack, and Wallaby, we also provide a set of testing helpers that help you:
+
+### Make fixtures:
+![fixtures](https://i.imgur.com/ettoUME.png)
+### Compare DOM:
+![compare-dom](https://i.imgur.com/pjGezjL.png)
+### Make sure batched updates are finished:
+![timing](https://i.imgur.com/iyE0IKf.png)
```

Comment:
probably leave that out or rephrase it as this is just the next frame... and many webcomponents are now rendering async so that might not be enough...

Reply:
How should I rephrase?

### Handle asynchronous updates:
### Manage timing:

?